src/DynamoCoreWpf/ViewModels/Core/NoteViewModel.cs

```diff
@@ -344,10 +344,13 @@ private void SubscribeToPinnedNode()
             Model.PinnedNode.PropertyChanged += PinnedNodeModel_PropertyChanged;
             PinnedNode.PropertyChanged += PinnedNodeViewModel_PropertyChanged;
 
-            // Subscribe to pinnedNode.RequestSelection (fires before node is selected)
+            // Subscribe to PinnedNode.Selected (fires before node is selected)
             // so that this note is added to the selection
             PinnedNode.Selected += PinnedNodeViewModel_OnPinnedNodeSelected;
```

Comment:
Maybe outside of this task but can you check if this is unsubscribed correctly?

Reply:
When reading the code from unsubscribe function, seems two events about select are misaligned, would you correct this case?

Reply:
Updated